modules/infinite-scroll/infinity.php

```diff
@@ -1382,8 +1386,13 @@ function query() {
 				$wrapper_classes = is_string( self::get_settings()->wrapper ) ? self::get_settings()->wrapper : 'infinite-wrap';
 				$wrapper_classes .= ' infinite-view-' . $page;
 				$wrapper_classes = trim( $wrapper_classes );
+				$aria_label = sprintf(
+					/* translators: %1$s is the page count */
+					__( 'Page: %1$d.', 'jetpack' ),
+					$page
+				);
 
-				$results['html'] = '<div class="' . esc_attr( $wrapper_classes ) . '" id="infinite-view-' . $page . '" data-page-num="' . $page . '">' . $results['html'] . '</div>';
+				$results['html'] = '<div class="' . esc_attr( $wrapper_classes ) . '" id="infinite-view-' . $page . '" data-page-num="' . $page . '" tabindex="0" aria-label="' . esc_attr( $aria_label ) . '">' . $results['html'] . '</div>';
```

Comment:
Could you try [`role="region"` or a `<section>` here?](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/region.html) That should group things for a screen reader user without creating an unexpected tab-stop for a keyboard-only user.

Reply:
Added, this is a nice improvement.